test/lib/user.test.js

```diff
@@ -242,6 +242,20 @@ describe('User Model', () => {
         });
     });
 
+    describe('removeSettings', () => {
+        beforeEach(() => {
+            user.settings = {};
```

Comment:
Can you add some data other than `{}`?
If the old data were `{}`, and the expected new data is also `{}`, then we dont really know if `removeSettings` worked or not.

I have sample data like

```
const userSettingsMock = {
  1018240: {
    showPRJobs: true
  },
  1048190: {
      showPRJobs: false
  },
  displayJobNameLength: 30
};
```